src/components/Slider/Slider.vue

```diff
@@ -93,7 +134,7 @@ export default {
       handler() {
         this.$nextTick(() => {
           this.configureTooltip();
-          if (this.$refs['value-input'].textContent !== this.sliderVal) {
+          if (this.$refs['value-input'] && this.$refs['value-input'].textContent !== this.sliderVal) {
```

Comment:
This validation is extensive, I recommend creating variables to make it more readable.